apps/mull-ui/src/app/pages/event-page/event-page-info/event-page-info.tsx

```diff
@@ -59,26 +51,18 @@ export const EventPageInfo = ({
   return (
     <div className={`event-page-info-container ${className}`} style={style}>
       <div className="info-row">
-        {/* TODO: Currently using placeholder for the profile picture. The US will need to actually fetch image from media server */}
-        <img
-          src="https://i.pinimg.com/originals/0c/3b/3a/0c3b3adb1a7530892e55ef36d3be6cb8.png"
-          className="event-page-icon"
-          alt="Event"
-        ></img>
-        {/* TODO: Remove placeholder text once users are implemented */}
+        <img src={avatarUrl(event.host as User)} className="event-page-icon" alt="Event"></img>
```

Comment:
This is a rectangle for me. Can you make it circular like the other user-profile images. You will need to add the following styling to this element

```
border-radius: 50%;
object-fit: cover;
```